templates/cluster-template-kube-vip.yaml

```diff
@@ -224,6 +224,7 @@ metadata:
   name: ${CLUSTER_NAME}
 spec:
   facility: ${FACILITY}
+  metro: ${METRO}
```

Comment:
[Line 140 of this template](https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/538/files#diff-c094b19d687639893dcca22e7317dd71e12eca27ad9b554b6c3a73b28995fe6fR140) references `${FACILITY}`.  Should it be updated to also reference `${METRO}`?

Reply:
good call. updated all templates to use metros now.